modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -0,0 +1,602 @@
+    
+    ScreenSpaceRenderable* renderable = global::renderEngine->screenSpaceRenderable(id);
+    renderable->translate(glm::vec2(transX, transY), glm::vec2(startX, startY));
+
+    return 0;
+}
+
+int removeSelectedImageInBrowser(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 2, "lua::removeSelectedImageInBrowser");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    // Image index
+    const int i = ghoul::lua::value<int>(L, 1);
+    // Get browser
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    const ImageData& image = module->getWwtDataHandler()->getImage(i);
+        
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->removeSelectedImage(i);
+    }
+
+    return 0;
+}
+
+int setEquatorialAim(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 3, "lua::setEquatorialAim");
+    // Browser id
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    double ra = ghoul::lua::value<double>(L, 1);
+    double dec = ghoul::lua::value<double>(L, 1);
+
+    // Get module
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->setEquatorialAim(glm::dvec2(ra, dec));
+    }
+
+    return 0;
+}
+
+int setVerticalFov(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 2, "lua::setVerticalFov");
+    // Browser id
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    float vfov = ghoul::lua::value<float>(L, 1);
+
+    // Get module
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->setVerticalFov(vfov);
+    }
+
+    return 0;
+}
+
+int setBorderColor(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 4, "lua::setBorderColor");
+    // Browser id
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    int r = ghoul::lua::value<int>(L, 1);
+    int g = ghoul::lua::value<int>(L, 1);
+    int b = ghoul::lua::value<int>(L, 1);
+    glm::ivec3 color{ r, g, b };
+    // Get module
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->setBorderColor(color);
+    }
+
+    return 0;
+}
+
+int setScreenSpaceSize(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 3, "lua::setScreenSpaceSize");
+    // Browser id
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    float sizeX = ghoul::lua::value<float>(L, 1);
+    float sizeY = ghoul::lua::value<float>(L, 1);
+
+    // Get module
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->setScreenSpaceSize(glm::vec2(sizeX, sizeY));
+    }
+    return 0;
+}
+	
+}
```

Comment:
Closing brace should have the namespace on it as well:
`// namespace openspace::skybrowser::luafunctions`